src/site.rs

```diff
@@ -223,41 +225,37 @@ impl Site {
         html
     }
 
+    /// Copy static file to public directory.
+    pub fn copy_static_file<P: AsRef<Path>>(&self, path: P) -> Result<()> {
+        let relative_path = path.as_ref().strip_prefix(&self.static_path).unwrap();
+        let target_path = self.output_path.join(relative_path);
+        copy(path.as_ref(), &target_path)?;
+        Ok(())
```

Comment:
does it work if you are copying a folder that didn't exist? Might need to test manually what happens and create the folders manually by splitting the path components if necessary